### PR TITLE
Removing the try catch, adding check

### PR DESCRIPTION
**Pull request by itsacoderepo**

### CODE REVIEW

File: joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/mips/MipsFunctionPass.scala [hunk 1]
```diff
@@ -29,64 +29,68 @@ class MipsFunctionPass(currentProgram: Program,
...
-                .flatMap(x => Option(x.getSymbol))
-              if (symbol.isDefined) {
-                name = symbol.get.getName
-              }
+    val target = instruction.getPcode.toList.head.getSeqnum.getTarget
+    //jal 0x00000000 and some jalr t9
+    if (highFunction
+          .getPcodeOps(target)
+          .asScala
+          .toList
```
Comment (johannescoetzee) [inline joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/mips/MipsFunctionPass.scala:37]: You can extract this into a `val pCodeOps = highFunction.getPcodeOps(target).asScala.toList` to avoid repeating it below.

File: joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/mips/MipsFunctionPass.scala [hunk 2]
```diff
@@ -117,7 +121,7 @@ class MipsFunctionPass(currentProgram: Program,
                                      instruction.getMinAddress.getOffsetAsBigInteger.intValue)
             connectCallToArgument(instructionNode, node)
           case _ =>
-            logger.warn(
+            println(
```
Comment (johannescoetzee) [inline joern-cli/frontends/ghidra2cpg/src/main/scala/io/joern/ghidra2cpg/passes/mips/MipsFunctionPass.scala:124]: Why did you change this from using the logger to just writing to console?

**Review by johannescoetzee — CHANGES_REQUESTED** [review]

**Review by johannescoetzee — APPROVED** [review]

**Review by fabsx00 — APPROVED** [review]
